modules/ROOT/pages/to-download-an-asset.adoc

```diff
@@ -1,21 +1,32 @@
-= To Download an Asset
+= Download an Asset
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
+:imagesdir: ../assets/images/
 
 Download lets you download a copy of the asset, or in the case of a Custom asset, the optional file attached to the asset when it was created.
 
+After creating a REST asset, refresh the page before clicking Download.
```

Comment:
We should delete this and explain that the Mule3 and Mule4 connectors are generated in background. A refresh does not guarantee that the corresponding download button will appear.

Reply:
Good point. Fixing now.